browser/pages/account/controller.js

```diff
@@ -22,7 +23,14 @@ class AccountController {
     this.authFailed = false;
     this.tandcNotSigned = false;
     this.isLoginBtnClicked = false;
-    this.rememberMe = false;
+    this.rememberMe = TokenStore.getStatus();
+    if (this.rememberMe === false) {
+      let dataFilePath = path.join(Platform.localAppData(), 'settings.json');
+      if(fs.existsSync(dataFilePath)) {
+        TokenStore.deleteItem('login', this.installerDataSvc.username);
+        rimraf.sync(dataFilePath);
```

Comment:
@sudhirverma The `settings.json` file is getting deleted once the user has unchecked the `Remember Me` field, but the username and password field still shows up in the entries for that iteration of the application.